src/command/Unzip.re

```diff
@@ -0,0 +1,10 @@
+open Bindings;
+
+let run = (~p=?, file) =>
+  ChildProcess.exec(
+    {j|unzip $file|j},
```

Comment:
Can we always assume this binary is present?

Reply:
No, we cant. Thanks for catching this. Ideally, the entire Setup.re could use `module Cmd` (from Toolchain.re), but in the interest of time, I have kept it as it is

Reply:
In practice, however, we have found `unzip` on all three platforms

Reply:
Seems fine with me. If we are confident that this binary is available, then it isn't a problem.k